com.unity.multiplayer.mlapi/Runtime/Core/NetworkManager.cs

```diff
@@ -35,6 +35,9 @@ public class NetworkManager : MonoBehaviour, INetworkUpdateSystem, IProfilableTr
 #if UNITY_2020_2_OR_NEWER
         // RuntimeAccessModifiersILPP will make this `public`
         internal static readonly Dictionary<uint, Action<NetworkBehaviour, NetworkSerializer, __RpcParams>> __ntable = new Dictionary<uint, Action<NetworkBehaviour, NetworkSerializer, __RpcParams>>();
+        // IF TOOLS
+        // For debug and profiling purpose only
```

Comment:
Then this should only happen in debug builds, yes?

Reply:
@MFatihMAR Wanted to put warnings all over the place. In the next sprint, we will surround those with the tools define. We just need to merge our packages before we have this define in place. Those are more for a reminder to update those place. Or would you rather we wait for that to be done before we merge this PR?

Reply:
I was talking about `#if DEVELOPMENT_BUILD || UNITY_EDITOR` so that it's compiled out of release builds.

Reply:
ditto here, also put a similar comment above.
I thought it wouldn't be just `#if DEVELOPMENT_BUILD || UNITY_EDITOR` wrapped into `#if OUR_DEF_TOOLS` macro but rather yet another macro to be defined in player/build settings to enable/disable, therefore I suggested keeping it enabled with comment surroundings and wrap with that macro when it lands.
if that's not the case, then yeah, we should wait for other things to merge this into the `develop` branch (as per my previous comment above).

Reply:
Like my other comment, I think we should remove the comment rather than add conditionals here. We shouldn't prescribe the use case in the implementation, we should just make the implementation work. I can't see much reason for why "Internally lookup rpc name from the uint identifier at runtime" needs to have "only accessible from debug code" as a requirement.

Reply:
agreed. since it's internal-only, it would not cause big problems, I wouldn't say the same if it was public to devs though.

Reply:
Sound good. I'll remove the comments and not put any conditionals.